srcpkgs/samba/patches/musl_uintptr.patch

```diff
@@ -0,0 +1,35 @@
+commit f81e5b71ce78f33250347914dacc75c8463bf102
+Author: Breno Leitao <breno.leitao@gmail.com>
+Date:   Wed Mar 29 15:22:38 2017 -0300
+
+    include: Check for previous declaration of uintptr_t
+    
+    Adding a extra check before declaring uintptr_t. Currently musl uses
+    macro __DEFINED_uintptr_t once it defines uintptr_t type. Checking
+    this macro before defining it, and, defining it when uintptr_t is
+    defined.
+    
+    Signed-off-by: Breno Leitao <breno.leitao@gmail.com>
+
```

Comment:
Where did you find this patch? :( 

Just adding `#include <stdint.h>` adds the definition of `uintptr_t`....

Reply:
These patches were cargo-culted straight from Alpine.

Reply:
Argh :/

Well, they ought to work. If I find the energy to touch samba I can try to refactor some.